src/otx/config/train.yaml

```diff
@@ -0,0 +1,15 @@
+defaults:
+  - base_config
```

Comment:
What is base_config? and can we make `base_config`` a yaml?

Reply:
It's schema and comes from https://github.com/openvinotoolkit/training_extensions/blob/b8d8654ac4e846af0d0c4f1b0e971c5332ff6932/src/otx/core/config/__init__.py#L31. Please see https://hydra.cc/docs/tutorials/structured_config/schema/

Reply:
And why do we need this?

Reply:
Did you read the hyperlink I attached? You can validate whether the input configuration follows the schema or not.